docs/addons/karpenter.md

```diff
@@ -213,35 +241,6 @@ Karpenter, starting from the OCI registry versions, will untar the files under `
 
 ## Upgrade Path
 
-1. Using an older version of the Karptner add-on, you may notice the difference in the "provisionerSpecs" property:
-
-```
-provisionerSpecs: {
-    'node.kubernetes.io/instance-type': ['m5.2xlarge'],
-    'topology.kubernetes.io/zone': ['us-east-1c'],
-    'kubernetes.io/arch': ['amd64','arm64'],
-    'karpenter.sh/capacity-type': ['spot','on-demand'],
-  },
-```
-
-This now changes to the "requirement" property:
-
-```
-requirements: [
-      { key: 'node.kubernetes.io/instance-type', op: 'In', vals: ['m5.2xlarge'] },
-      { key: 'topology.kubernetes.io/zone', op: 'NotIn', vals: ['us-west-2c']},
-      { key: 'kubernetes.io/arch', op: 'In', vals: ['amd64','arm64']},
-      { key: 'karpenter.sh/capacity-type', op: 'In', vals: ['spot','on-demand']},
-  ],
-```
-
-The property is changed to align with the naming convention of the provisioner, and to allow multiple operators (In vs NotIn). The values correspond similarly between the two, with type change being the only difference.
-
-2. Certain upgrades require reapplying the CRDs since Helm does not maintain the lifecycle of CRDs. Please see the [official documentations](hhttps://karpenter.sh/docs/upgrading/upgrade-guide/) for details.
-
-3. Starting with v0.17.0, Karpenter's Helm chart package is stored in OCI (Open Container Initiative) registry. With this change, [charts.karpenter.sh](https://charts.karpenter.sh/) is no longer updated to preserve older versions. You have to adjust for the following:
-
-* The full URL needs to be present (including 'oci://').
-* You need to append a `v` to the version number (i.e. v0.17.0, not 0.17.0)
+1. Certain upgrades require reapplying the CRDs since Helm does not maintain the lifecycle of CRDs. Please see the [official documentations](https://karpenter.sh/docs/upgrading/upgrade-guide/) for details.
```

Comment:
is this something that blueprints can handle seemlessly. Please see my comment on the next paragraph as well.

Reply:
This should be taken care of. I can remove this wording.